### PR TITLE
Add sane doc reports script

### PR DESCRIPTION
**Pull request by agamm**
## Status
Ready

## Related Issues
https://github.com/demisto/etc/issues/17374

## Description
Adds the automation for the docx report generation. 
This is an internal automation that is invoked from the server.

## Related PRs
https://github.com/demisto/server/pull/12889
https://github.com/demisto/dockerfiles/pull/67

## Required version of Demisto
x.x.x

## Does it break backward compatibility?
   - No

## Must have
- [ ] Code Review

### CODE REVIEW

File: Scripts/SaneDocReport/SaneDocReport.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+args:
+- default: true
+  description: sane report json file contents base64
+  isArray: false
+  name: saneJsonBase64
```
Comment (ronykoz) [inline Scripts/SaneDocReport/SaneDocReport.yml]: sane-report-encoded

File: Scripts/SaneDocReport/SaneDocReport.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    with open('sane.json', 'wb') as f:
+        f.write(base64.b64decode(sane_json_b64))
+
+    run('sane.json', 'out.docx')
+
+    with open('out.docx', 'rb') as f:
+        encoded = base64.b64encode(f.read()).decode("utf-8", "ignore")
+        out = {'data': 'encoded'}
+        ec_data = createContext(data=out)
+        ec = {'saneDocReport.data': ec_data}
```
Comment (ronykoz) [inline Scripts/SaneDocReport/SaneDocReport.py]: I guess you should use it

File: Scripts/SaneDocReport/SaneDocReport.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+        return_outputs(readable_output='Successfully generated docx',
```
Comment (ronykoz) [inline Scripts/SaneDocReport/SaneDocReport.py]: why not to add the file as file entry?

File: Scripts/SaneDocReport/SaneDocReport.py (hunk 2 above)
Comment (agamm) [reply Scripts/SaneDocReport/SaneDocReport.py]: I am using it from the server, with:
```
module := modules.Registry.Get(domain.CustomScriptsModule)
entries, err := module.Execute(scriptName, saneJSONPathArg, context)
// ...
contents := entries[0].Contents
```

File: Scripts/SaneDocReport/SaneDocReport.py (hunk 3 above)
Comment (agamm) [reply Scripts/SaneDocReport/SaneDocReport.py]: Is this still a valid point after my last comment?

**Comment by agamm** [comment]
@ronykoz can you recheck to make sure everything that got changed is good?

File: Scripts/SaneDocReport/SaneDocReport.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+import base64
+from sane_doc_reports.main import run
+
+try:
+    sane_json_b64 = demisto.args().get('sane_docx_report_base64', '').encode(
+        "utf-8")
+    with open('sane.json', 'wb') as f:
+        f.write(base64.b64decode(sane_json_b64))
+
+    run('sane.json', 'out.docx')
```
Comment (dorsha) [inline Scripts/SaneDocReport/SaneDocReport.py]: let's clean after

**Comment by agamm** [comment]
@dorsha updated.

**Comment by dorsha** [comment]
Cool, merge

**Pull request merged**